decidim-core/app/models/decidim/attachment.rb

```diff
@@ -95,6 +114,11 @@ def set_content_type_and_size
       self.file_size = file.byte_size
     end
 
+    def set_link_content_type_and_size
+      self.content_type = "decidim/link"
```

Comment:
According to [this stackoverflow answer](https://stackoverflow.com/a/22754385), we can use  `text/uri-list` if we end the link with a CRLF (`\r\n`) 

```suggestion
      self.content_type = "text/uri-list"
```